File: JavaScript/Easy/SumOfDigitsOfStringAfterConvert.js
/**
 * @param {string} s
 * @param {number} k
 * @return {number}
 */
var getLucky = function (s, k) {
  let result = "";
  for (let i = 0; i < s.length; i++) {
    result += s.charCodeAt(i) - 96;
  }

  while (k--) {
    let sum = 0;
    for (let i = 0; i < result.length; i++) {
      sum += +result[i];
    }
    result = String(sum);
  }

  return +result;
};

File: Python/Easy/SumOfDigitsOfStringAfterConvert.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        result = "".join(str(ord(c) - 96) for c in s)

        for _ in range(k):
            result = str(sum(int(digit) for digit in result))

        return int(result)
